alerts.tf

```diff
@@ -0,0 +1,91 @@
+module "cw_alerts" {
+  count = var.alarms.enabled ? 1 : 0
+
+  source  = "dasmeta/monitoring/aws//modules/alerts"
+  version = "1.3.5"
+
+  sns_topic = var.alarms.sns_topic
+
+  alerts = [
+    {
+      name   = "DocumentDB ${var.name} CPUUtilization"
+      source = "AWS/RDS/CPUUtilization"
```

Comment:
Is AWS/RDS/CPUUtilization the right namespace? Or should it be AWS/DocDB/CPUUtilization ?